src/qml/qgismobileapp.qml

```diff
@@ -20,6 +20,7 @@ import QtQuick.Controls 1.4 as Controls
 import QtQuick.Controls 2.4
 import QtQuick.Dialogs 1.2
 import QtGraphicalEffects 1.0
+import Qt.labs.settings 1.0 as LabSettings
```

Comment:
Are we ok to use labs?

Reply:
We already use it for the settings https://github.com/opengisch/QField/blob/master/src/qml/QFieldSettings.qml#L3